src/main/java/de/refactoringbot/refactoring/supportedrefactorings/ReorderModifier.java

```diff
@@ -54,7 +54,8 @@ public String performRefactoring(BotIssue issue, GitConfiguration gitConfig) thr
 			method.setModifiers(new NodeList<Modifier>());
 			method.setModifiers(modifiersInCorrectOrder);
 		} else {
-			throw new BotRefactoringException("Could not find method or field declaration at the given line.");
+			throw new BotRefactoringException(
+					"Could not find method or field declaration at the given line! Please check if you placed your comment on a method or field declaration.");
```

Comment:
Same issue with the second part of the message as already mentioned in a previous comment.